Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -15,37 +16,61 @@ namespace Leap.Unity {
    */
   public class HandPool :
     HandFactory {
-
+    public bool EnforceHandedness = false;
```

Comment:
should be private with a SerializeField, if the value needs to be accessed outside this class, a separate public getter can be added.